### DISCUSSION TITLE
Error: Client network socket disconnected before secure TLS connection was established

### DISCUSSION THREAD

**Discussion by aafnnp** [uicrooks/shopify-theme-lab#96] [Q&A]
Error: Client network socket disconnected before secure TLS connection was established

what's wrong?

**Comment by sergejcodes** [comment]
There needs to be more narrowing down to figure out the problem, can you provide some additional information about:

- your browser and version (Chrome / Firefox / etc.)
- your System (Windows / Mac / etc.)
- your Node.js version
- when are you getting this error: in Browser or in the Terminal during an npm script (if so which script/task is it)?
